ycms/static/src/css/style.scss

```
@@ -179,7 +192,7 @@ a,
 input[type="submit"] {
     // Normal buttons
     &.btn {
-        @apply font-bold rounded inline-block bg-primary text-gray-700 py-3 px-4 border-2 border-primary;
+        @apply font-bold rounded inline-block bg-primary text-gray-700 py-3 px-4 border-2 border-primary dark:bg-gray-800 dark:text-primary dark:hover:bg-primary dark:hover:text-gray-800;
```

```suggestion
        @apply font-bold rounded inline-block bg-primary text-gray-700 py-3 px-4 border-2 border-primary dark:bg-gray-800 dark:text-primary;
```

My bad - `dark:hover:bg-primary dark:hover:text-gray-800` needs to go in the `&:hover{}` section a couple of line below, otherwise these styles are being overwritten.